actor/src/lib.rs

```diff
@@ -259,6 +287,10 @@ where
         self.do_send(m).await
     }
 
+    fn send_sync(&self, m: M) {
```

Comment:
Not  a huge fan of `*_sync` nomenclature. Have you considered `send_blocking`? Namespacing the functions to a `synchronous` module?

Reply:
A module seems like it would be quite a hassle. I agree _sync isn't ideal, but blocking also implies some kind of wait when the point of this function is that it returns nearly instantly. Maybe in the future we could have it be just send? The async code should be able to use send_sync.

Reply:
I'd prefer a future where async can live with sync. Feel free to resolve.